ui/src/extractedTranslations/fr/translation.json

```diff
@@ -209,20 +209,20 @@
   },
   "sidebar": {
     "album": {
-      "title_placeholder": null
+      "title_placeholder": "Titre de l'Album"
     },
     "download": {
       "filesize": {
         "byte": "{{count}} Octets",
         "byte_plural": "{{count}} Octets",
         "giga_byte": "{{count}} Go",
-        "giga_byte_plural": null,
+        "giga_byte_plural": "Go",
         "kilo_byte": "{{count}} Ko",
-        "kilo_byte_plural": null,
+        "kilo_byte_plural": "Ko",
         "mega_byte": "{{count}} Mo",
-        "mega_byte_plural": null,
+        "mega_byte_plural": "Mo",
         "tera_byte": "{{count}} To",
-        "tera_byte_plural": null
+        "tera_byte_plural": "To"
```

Comment:
The {{count}} is a special placeholder where the number will be inserted.
Please revert this